{"author":{"id":"3998add589f6a09cf4bdf0e85e5481d2c48b8a757edd699a79a0c929ab5005db"},"ops":[{"type":3,"timestamp":1658755134,"nonce":"CZrwBhRPMVrqVxFi43RE0RjptfY=","metadata":{"github-id":"IC_kwDOCGKeh85HK5U-","github-url":"https://github.com/git-bug/git-bug/issues/653#issuecomment-1194038590"},"message":"https://github.com/MichaelMure/git-bug/pull/835 brought to light the bug that likely cause this. Somewhere in the code an EditCommentOperation was created, the Id() was taken to register the change in the timeline, then metadata were added to that operation, changing the data and the Id. A newly created assertion failed in that branch, fix pending.","files":null},{"type":4,"timestamp":1661259698,"nonce":"9O1UZVyQusuvP1AK+wPPmsAL52s=","metadata":{"github-id":"CE_lADOCGKeh84zropIzwAAAAGvpMa_"},"status":2}]}